frontend/src/modules/conversation/config/filters/dateStarted/options.ts

```diff
@@ -1,28 +1,29 @@
 import { SelectFilterOptionGroup } from '@/shared/modules/filters/types/filterTypes/SelectFilterConfig';
 import moment from 'moment';
 
+// Days period is -1 to include today's data
 const options: SelectFilterOptionGroup[] = [
   {
     options: [
       {
         label: 'Last 24 hours',
-        value: moment().subtract(24, 'hour').format('YYYY-MM-DD'),
+        value: moment().utc().subtract(24, 'hour').format('YYYY-MM-DD'),
       },
       {
         label: 'Last 7 days',
-        value: moment().subtract(7, 'day').format('YYYY-MM-DD'),
+        value: moment().utc().subtract(6, 'day').format('YYYY-MM-DD'),
```

Comment:
why the number of subtracted days has been decreased from 7 to 6?

Reply:
Because our last 7 days, to match the dashboard are the last 6 days + today. Since those filters are using gte operator, to match the dashboard requests I had to decrease the count by one, since the first day also counts for the filter and today makes the last day